test/e2e/networkpolicy_test.go

```diff
@@ -270,6 +271,165 @@ func TestDefaultDenyEgressPolicy(t *testing.T) {
 	}
 }
 
+// TestEgressToServerInCIDRBlock is a duplicate case as "should allow egress access to server in CIDR block [Feature:NetworkPolicy]",
+// while a bug exists in TOT release. This should be deleted when upstream is updated.
+// https://github.com/kubernetes/kubernetes/blob/master/test/e2e/network/network_policy.go#L1373
```

Comment:
please use perma links

Reply:
Used link for `v1.20.0-alpha.0`. Also links in the PR description and commit message are provided.

Yes, code in latest release doesn't consider netmask for IPv6 environment.